### DISCUSSION TITLE
Dependencies publicly available in github?

### DISCUSSION THREAD

**Discussion by kainutom** [helsenorge/refero#80] [General]
Intresting project!
Do you have plans to make @helsenorge/* source code repositories publicly available?
I mean these ones:
[@helsenorge/core-utils](https://www.npmjs.com/package/@helsenorge/core-utils)
[@helsenorge/file-upload](https://www.npmjs.com/package/@helsenorge/file-upload)
[@helsenorge/form](https://www.npmjs.com/package/@helsenorge/form)
[@helsenorge/date-time](https://www.npmjs.com/package/@helsenorge/date-time)
[@helsenorge/autosuggest](https://www.npmjs.com/package/@helsenorge/autosuggest)
[@helsenorge/designsystem-react](https://www.npmjs.com/package/@helsenorge/designsystem-react)

**Comment by kennethmyhra** [comment]
There is ongoing work to open-source these components, but I do not have a set timeline for this yet. Soonish though:)

**Reply by kajami** [reply]
Hello,

Nice project you got here! Are there any updates regarding the timeline for open-sourcing these components?

[@helsenorge/core-utils](https://www.npmjs.com/package/@helsenorge/core-utils)
[@helsenorge/file-upload](https://www.npmjs.com/package/@helsenorge/file-upload)
[@helsenorge/form](https://www.npmjs.com/package/@helsenorge/form)
[@helsenorge/date-time](https://www.npmjs.com/package/@helsenorge/date-time)
[@helsenorge/autosuggest](https://www.npmjs.com/package/@helsenorge/autosuggest)
[@helsenorge/designsystem-react](https://www.npmjs.com/package/@helsenorge/designsystem-react)

**Reply by kennethmyhra** [reply]
@kajami It's in the backlog for the team that's responsible for those components, but work has not yet been started

**Reply by larskrj** [reply]
@kajami: We have now open sourced the code for these packages. The code can be found here: https://github.com/helsenorge/designsystem and https://github.com/helsenorge/core-frontend-packages

**Comment by ruberino** [comment]
thiese are now available as npm packages, form is outdated and is no longer used
